halo/app/start.go

```diff
@@ -204,8 +246,28 @@ func Start(ctx context.Context, cfg Config) (<-chan error, func(context.Context)
 		return nil, nil, err
 	}
 
-	go monitorCometForever(ctx, cfg.Network, rpcClient, cmtNode.ConsensusReactor().WaitSync, cfg.DataDir())
-	go monitorEVMForever(ctx, cfg, engineCl)
+	readiness := &ReadyResponse{}
+	server := &http.Server{
+		Addr:              ":25560",
```

Comment:
don't hardcode this, use the existing cometBFT config to read the value

Reply:
also, don't this clash with cometBFT internal prometheus port like mentioned in the issue?

Reply:
Btw, the CometBFT Prometheus port is `26660` so there be no conflicts with the port suggested in the issue (`25560`) but the real question what is the reason for suggesting disabling the default prometheus server? We can have more than one server, each serving its own thing, right?

Reply:
I think I figured it out.